app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/TableRow/index.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import InputWidget from 'components/WranglerGrid/SelectColumnPanel/DataTable/InputWidgets';
+import DataQualityCircularProgressBar from 'components/common/DataQualityCircularProgressBar';
+import { ITableRowProps } from 'components/WranglerGrid/SelectColumnPanel/DataTable/types';
+import { TableCellText } from 'components/common/TypographyText';
+import { TableRow, TableCell } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import styled from 'styled-components';
+
+const SelectColumnTableRow = styled(TableRow)`
+  font-weight: 400;
```

Comment:
I see lot of common styles between the styled components here
plz make sure we are combining them

Reply:
Done